addon/components/simple-input.js

```diff
@@ -3,7 +3,7 @@ import humanize from "ember-form-builder/utilities/humanize";
 import guessType from "ember-form-builder/utilities/guess-type";
 
 const extension = {
-  i18n: Ember.inject.service(),
+  translationService: Ember.inject.service(),
```

Comment:
How about naming this service `formBuilderTranslations` instead of `translationService`?